bsp4j/src/main/java/ch/epfl/scala/bsp4j/ScalaBuildServer.java

```diff
@@ -14,4 +14,7 @@
 
     @JsonRequest("buildTarget/scalaMainClasses")
     CompletableFuture<ScalaMainClassesResult> buildTargetScalaMainClasses(ScalaMainClassesParams params);
+
+    @JsonRequest("buildTarget/jvmTestEnvironment")
+    CompletableFuture<JvmEnvironmentResult> jvmTestEnvironment(JvmEnvironmentParams params);
```

Comment:
👍 I like the new name, I don't have a strong opinion on `jvmTestEnvironment` vs. `jvmEnvironment`